tests/js/client/communication/test-communication.js

```diff
@@ -578,14 +555,15 @@ function GenericAqlSetupPathSuite(type) {
     })();
     `;
 
-    let args = ['--javascript.execute-string', cmd];
+    let args = {'javascript.execute-string': cmd};
     let pid = runShell(args, key);
     debug("started client with key '" + key + "', pid " + pid + ", args: " + JSON.stringify(args));
     return { key, pid };
   };
 
   const singleRun = (tests) => {
     let clients = [];
+    reconnectRetry(primaryEndpoint, "_system", "root", "");
```

Comment:
if the helper methods do this reconnectRetry internally this Is not required at all